### ISSUE TITLE
As a composer application developer I want to start with the same sample application that supports both HLF V0.6 and HLF V1.0 

### ISSUE THREAD

**Issue by fenglianxu** [hyperledger/composer#771]
Currently we have two getting started sample applications for HLF V0.6 and HLF V1.0 respectively.
The sample application project itself should be independent from the underneath Fabric runtime environment. 

@mbwhite @dselman and myself had an initial discussion to move this script to the composer-cli project but this solution has some drawbacks:

1. This needs to test existing project composer-cli which may cause some other issues e.g. poor performance on both travis build and setting up the fabric runtime. 
2. When a new version of Fabric is released, we will make changes to the composer-cli project again.

@dselman @sstone1 and myself had another discussion today to move the script into a separate package.
## Expected Behavior

The scripts setting up the underneath runtime between two versions should be merged into one project. This script project should be in a separate project. Any future runtime change would only need to change this script project without impacting other existing project. 

## Steps to Reproduce

1.
2.
3.
4.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/fabric-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/fabric-composer)

## Your Environment

* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

**Comment by dselman** [comment]
Basically we're going to create a GitHub repo called something like 'fabric-dev-servers' with hlfv1 and hlfv06 directories. Within each dir we will put the config for the respective dev server. These are completely independent of Composer, and we will work with the Fabric team to get the repo checked in and managed by the Fabric team. Let's show them what we have and discuss on the call on Tues. We will update our install and getting started docs to follow a more "classical" approach: 1/ Install the server (git clone). 2/ Start the sever 3/ Install Composer 4/ Run the Getting Started.

**Comment by mbwhite** [comment]
I'm creating these locally on my machine at present.

**Comment by dselman** [comment]
Thanks. It also gives us the opportunity to have a nice README.md that describes the config for each server, options etc.

**Comment by jdockter** [comment]
For v1 will that also include creating and joining channels, couchDB config...other items like orderer and CA config, system chain code, etc...  Just be good to know of what can and can't be done with this new repo compared to what Fabric recommends in their getting started here, http://hyperledger-fabric.readthedocs.io/en/latest/getting_startedv2.html

**Comment by mbwhite** [comment]
An updated set of Getting Started Instructions are listed at 
 https://github.com/mbwhite/documents/blob/master/NEW-GettingStarted.md

The current state is 
* that Getting Started repository will be used for both v0.6 and v1.0 Fabric versions. 
* composer-cli does not refer to Fabric
* A new repo is defined that contains scripts based on the Fabric's Sys Tests that start up the Fabrics instances. This only mentions the composer connection profiles that are needed to be created. 
https://github.com/mbwhite/fabric-dev-servers

**Comment by mbwhite** [comment]
- [x] The fabric-dev-tools are now part of the composer-tools repo, these are in the main repo.
- [] Currently waiting on the travis build to be enabled so the internal zip file can be built and the scripts tested in travis
- [] composer-sample-applications/getting-started is ready in my fork, and with updated readmes that use the new scripts. This can be merged in - but this affects the documentation etc.  

/cc @EdProsser I'll take your guidance as when is best to submit the pull request to get this code into the main repos.
/cc FYI @davidkel @dselman @nklincoln
